Images, improvements and boxes

jghomel opened the pull request:
added constants component to store icons
added icons
added iconWrapper component to style components
removed spaces and such in a number of components
added a nice box to app so that fused decks appear more clearly added boxstyles.js which tries to create a component so we can have multiple recyclable box styles, but didn't figure out how to use props to call a specific style. Included so you can help

jghomel committed:
added constants component to store icons
added icons
added iconWrapper component to style components
removed spaces and such in a number of components
added a nice box to app so that fused decks appear more clearly
added boxstyles.js which tries to create a component so we can have multiple recyclable box styles, but didn't figure out how to use props to
call a specific style. Included so you can help

lavoiejp-sonder commented on src/App.js:
```diff
@@ -29,16 +31,22 @@ export function App() {
-
-        })}
-      </List>
+      <Box sx={{margin: 2, border: 2, width: "16%", borderColor: 'grey.500', bgcolor: 'background.paper', borderRadius: 6}} justifyContent="center">
```
makeStyles?

Honnêtement je suis pas certain si c'est nécessaire de tous les extraires, je te laisse gérer ça.

lavoiejp-sonder commented on src/App.js:
```diff
@@ -1,9 +1,11 @@
 import DeckTable from './components/DeckTable'
 import DeckLink from './components/DeckLink';
 import ButtonWrapper from './components/ButtonWrapper';
+import BananaImage from './components/IconWrapper';
```
je pense que BananaImage devrait être renommé. Aussi, tu n'as pas ajouté le fichier de constantes dans git. (Probablement pour ça  que ton build a pété

janphilippelavoie replied:
ca dit encore banane ici... Et y'a une grosse banane dans les images

lavoiejp-sonder commented on src/components/DeckTable.js:
```diff
@@ -24,15 +28,18 @@ export default function DeckTable(props) {
-    { id: "healthI", label: "Average Health (I)", number: true },
-    { id: "healthII", label: "Average Health (II)", number: true },
-    { id: "healthIII", label: "Average Health (III)", number: true },
+    { id: "attackI", label: headerIcons("Average Attack", Paths.Level1), number: true },
```
Le nom de la constante n'est pas assé spécifique, on aura surement d'autres sortes de path plus tard. ImagePaths serait plus approprié

jghomel committed:
added all missing files

Turns out git commit -a doesn't add files. Who knew!
Also cleaned names in deck table and removed bananaimage

jghomel committed:
Added files

Turns out git commit -a with git add -all doesn't send your files over. Who knew!
Also made changes to names in Constants.

janphilippelavoie commented on src/components/BoxStyles.js:
```diff
@@ -0,0 +1,24 @@
+    }
+});
+
+export default function BoxStyles(props) {
```
BoxWrapper pour garder la standardization de nos nom?

janphilippelavoie commented on src/components/BoxStyles.js:
```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+
+const useStyles = makeStyles({
+    SmallRoundedBox: {
```
la convention est camelCase ici.

jghomel committed:
Responding to comments in pull request
